docs/guides/wipe-environment.rst

```diff
@@ -0,0 +1,28 @@
+Wiping a Build/Environment
+==========================
+
+Sometimes it happen that your Builds start failing because the
+environment where the documentation is created is stale or
+broken. This could happen for a couple of different reasons like `pip`
+not upgrading a package properly or a corrupted cached Python package,
+for example.
+
+In any of these cases (and many others), the solution could be just
+wiping out the existing build/environment files and allow Read the
+Docs to create a new fresh one.
+
+Follow these steps to wipe the build/environment:
+
+* Go to **Versions**
+* Click on the **Edit** button of the version you want to wipe on the
+  right side of the page
+* Go to the bottom of the page and click the **wipe** link, next to
+  the "Save" button
+
+.. note::
+
+   By wiping the documentation build/environment, all the files
+   associated with it will be removed but not the documentation
+   itself.
```

Comment:
This is confusing. Perhaps:

By wiping the documentation build environment, all the `rst, md, & code` files
associated with it will be removed but not the HTML/PDF itself.

Or something. We should be really explicit with what is happening.

Reply:
I tried to make it more explicit regarding the type of files that are going to be deleted and also add a mention like "your documentation will still online" to make it clear that wiping an build environment won't destroy your docs.